### DISCUSSION TITLE
Which OCR solution to implement on our SaaS platform?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#56191] [Other Feature Feedback, Questions, & Ideas]
Hi, I’m Dennis, non-tech co-founder of Siphter. We want to add an OCR solution to our SaaS platform. This OCR should be able to convert text and images to specific fields, creating a summary.

The documents that will be uploaded are CIM (Confidential Information Memorandum) in PDF, management summaries in PDF, Powerpoint converted to PDF etc.

Our tech stack is PHP/ JS/ Laravel.

Which OCR solutions would you advise? And why?

**Comment by tuves** [comment]
Welcome to the GitHub Community, @Hamsterconnect, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Reply by ghost** [reply]
Thank you Tuves!
